CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# Changelog
```

Comment:
Should there be something else in here?

Reply:
@cdwilhelm thoughts. It looks like in policy_templates its empty also, it looks like we need to exclude the repo root in the rule .